tests/test_client.py

```diff
@@ -78,6 +83,36 @@ def test_destroy_documents(self):
             response = self.client.destroy_documents(self.engine_name, [id])
             self.assertEqual(response, expected_return)
 
+    def test_list_engines(self):
+        expected_return = [
+            { 'name': 'myawesomeengine' }
+        ]
+
+        with requests_mock.Mocker() as m:
+            url = "{}/{}".format(self.client.swiftype_session.base_url, 'engines')
+            m.register_uri('GET',
```

Comment:
i would write this as:
```
m.register_uri(
                'GET',
                url,
                additional_matcher=lambda x: x.text == '{"page": {"current": 10, "size": 2}}',
                json=expected_return,
                status_code=200
            )
```

and github isn't being nice, so: 
![deepinscreenshot_select-area_20180518150734](https://user-images.githubusercontent.com/4838266/40259957-38222476-5aad-11e8-8783-6ec6f21a29b8.png)

also using a lambda for the match requesting. That's a good way of having a one-off mini function

Reply:
Ah, very nice, updating this now.